src/coreclr/src/pal/inc/pal.h

```diff
@@ -1107,13 +1098,7 @@ OpenSemaphoreW(
     IN BOOL bInheritHandle,
     IN LPCWSTR lpName);
 
-#ifdef UNICODE
-#define CreateSemaphore CreateSemaphoreW
-#define CreateSemaphoreEx CreateSemaphoreExW
-#else
-#define CreateSemaphore CreateSemaphoreA
-#define CreateSemaphoreEx CreateSemaphoreExA
-#endif
+#define CreateSemaphore CreateSemaphoreExW
```

Comment:
This should be still `#define CreateSemaphoreEx CreateSemaphoreExW`. Otherwise, CreateSemaphore will take different number of arguments on Windows vs. Unix.

Reply:
Renamed to CreateSemaphoreEx and fixed up the tests which were using CreateSemaphore.